projects/pastanaga/src/lib/translate/translate.pipe.ts

```diff
@@ -0,0 +1,33 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import * as translateKeys from '../../../../../src/assets/i18n/en.json';
```

Comment:
relative path like that is not ok in pastanaga. `en.json` content should be provided in the caller module.